src/components/Button.tsx

```diff
@@ -194,6 +195,10 @@ function getColors(type: BtnTypes, disabled: boolean | undefined) {
       backgroundColor = colors.gray1
       borderColor = colors.gray2
       break
+    case BtnTypes.LABEL_PRIMARY:
```

Comment:
No, I didn't... I'll check with her if we are gonna reuse this pattern.

I did this to avoid more complexity... It seems that the [button decides the color of the text](https://github.com/valora-inc/wallet/blob/2f60f03cb3b15ce8f1340adc4c3d408651f462cd/src/components/Button.tsx#L121).
And then the only "pretty" way was [adding a new type here](https://github.com/valora-inc/wallet/blob/2f60f03cb3b15ce8f1340adc4c3d408651f462cd/src/components/Button.tsx#L164).
Even if we don't use that combination again, I think we should have a specific button type for it. Unless [this comment](https://github.com/valora-inc/wallet/blob/2f60f03cb3b15ce8f1340adc4c3d408651f462cd/src/components/Button.tsx#L120) is no longer true (or we explore another workaround for it).